src/components/search/no-results-message.vue

```diff
@@ -19,7 +18,7 @@
     > span {
       width: 100%;
 
-      .x-text {
+      span {
```

Comment:
You can replace this style if we enable the `wordBreak` core plugin in the tailwind config and we add the `x-break-words` class: https://tailwindcss.com/docs/word-break